widgets/binilla_widget.py

```diff
@@ -13,140 +13,177 @@
+                family=e_c.DEFAULT_FONT_FAMILY, size=e_c.DEFAULT_FONT_SIZE,
+                weight=e_c.DEFAULT_FONT_WEIGHT, slant=e_c.DEFAULT_FONT_SLANT),
+
+            console=font_config.FontConfig(
+                family=e_c.FIXED_FONT_FAMILY, size=e_c.FIXED_FONT_SIZE + 1,
+                weight=e_c.FIXED_FONT_WEIGHT, slant=e_c.FIXED_FONT_SLANT),
+
+            heading=font_config.FontConfig(
+                family=e_c.HEADING_FONT_FAMILY, size=e_c.HEADING_FONT_SIZE,
+                weight=e_c.HEADING_FONT_WEIGHT, slant=e_c.HEADING_FONT_SLANT),
+            heading_small=font_config.FontConfig(
+                family=e_c.HEADING_SMALL_FONT_FAMILY, size=e_c.HEADING_SMALL_FONT_SIZE,
+                weight=e_c.HEADING_SMALL_FONT_WEIGHT, slant=e_c.HEADING_SMALL_FONT_SLANT),
+            frame_title=font_config.FontConfig(
+                family=e_c.CONTAINER_TITLE_FONT_FAMILY, size=e_c.CONTAINER_TITLE_FONT_SIZE,
+                weight=e_c.CONTAINER_TITLE_FONT_WEIGHT, slant=e_c.CONTAINER_TITLE_FONT_SLANT),
+
+            comment=font_config.FontConfig(
+                family=e_c.COMMENT_FONT_FAMILY, size=e_c.COMMENT_FONT_SIZE,
+                weight=e_c.COMMENT_FONT_WEIGHT, slant=e_c.COMMENT_FONT_SLANT),
+            tooltip=font_config.FontConfig(
+                family=e_c.DEFAULT_FONT_FAMILY, size=e_c.DEFAULT_FONT_SIZE,
+                weight=e_c.DEFAULT_FONT_WEIGHT, slant=e_c.DEFAULT_FONT_SLANT),
+            )
+
+        # MISC
+        cls.title_width = e_c.TITLE_WIDTH
+        cls.enum_menu_width = e_c.ENUM_MENU_WIDTH
+        cls.scroll_menu_width = e_c.SCROLL_MENU_WIDTH
+        cls.min_entry_width = e_c.MIN_ENTRY_WIDTH
+        cls.textbox_height = e_c.TEXTBOX_HEIGHT
+        cls.textbox_width = e_c.TEXTBOX_WIDTH
+
+        cls.bool_frame_min_width = e_c.BOOL_FRAME_MIN_WIDTH
+        cls.bool_frame_min_height = e_c.BOOL_FRAME_MIN_HEIGHT
+        cls.bool_frame_max_width = e_c.BOOL_FRAME_MAX_WIDTH
+        cls.bool_frame_max_height = e_c.BOOL_FRAME_MAX_HEIGHT
+
+        cls.def_int_entry_width = e_c.DEF_INT_ENTRY_WIDTH
+        cls.def_float_entry_width = e_c.DEF_FLOAT_ENTRY_WIDTH
+        cls.def_string_entry_width = e_c.DEF_STRING_ENTRY_WIDTH
+
+        cls.max_int_entry_width = e_c.MAX_INT_ENTRY_WIDTH
+        cls.max_float_entry_width = e_c.MAX_FLOAT_ENTRY_WIDTH
+        cls.max_string_entry_width = e_c.MAX_STRING_ENTRY_WIDTH
+
+        cls.scroll_menu_max_width = e_c.SCROLL_MENU_MAX_WIDTH
+        cls.scroll_menu_max_height = e_c.SCROLL_MENU_MAX_HEIGHT
+
+        cls.can_scroll = False
+        cls.tooltip_string = None
+        cls.f_widget_parent = None
+        cls.disabled = False
+
+        cls.font_type = "default"  # the type of font to use
+        if dark:
+            cls.ttk_theme = "clam"
+        else:
+            cls.ttk_theme = "alt"
+
+        cls.read_traces = ()
+        cls.write_traces = ()
+        cls.undefine_traces = ()
+
+        cls._filedialog_style_fix = None

```

Comment:
Do not set _filedialog_style_fix to None. Let it be defined as None in BinillaWidget, and let it be properly initialized in the \_\_init\_\_ method. The _filedialog_style_fix should only ever be instantiated ONCE in the lifetime of binilla. That should fix fix_filedialog_style breaking, as you commented it out below.